reference/04-Patterns/synced-patterns.md

```diff
@@ -0,0 +1,152 @@
+---
+sidebar_position: 2
+---
+
+# Synced Patterns
```

Comment:
Just for historical context I would update the title to be Synced Patterns (Reusable Blocks)

Reply:
done!